frontend/src/services/authentication_handler_service/authentication_handler_service.ts

```diff
@@ -0,0 +1,71 @@
+import { BackendAuthenticationInterface } from "../backend_service_interface/backend_service_interface";
+import { CLIENT_ID } from "./authentication_constants";
+
+/** Error that occurs if sign in fails. */
+export class FailureToSignIn extends Error {
```

Comment:
nit: By convention, `FailureToSignInError`

Reply:
Thanks! Updated and pushed.